src/manifest.json

```diff
@@ -1,34 +1,27 @@
 {
-   "manifest_version": 2,
-    "name": "Transfer Send for Firefox",
-    "version": "1.8",
-    "description": "Facilitate file transfer with Firefox Send (https://send.firefox.com) while Keeping your Personal Information Private",
+    "manifest_version": 2,
+    "name": "Firefox Send Sidebar",
```

Comment:
I made some updates to the `name` and `description` fields. If you prefer your originals I am more than happy to revert.